### DISCUSSION TITLE
Show Viewed Files State In Left Tree View

### DISCUSSION THREAD

**Discussion by KoltonG** [community/community#59469] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hi GitHub Team!

Thank you for continuously improving the product, especially the revamp you've all done to the Pull Request experience! 

I've been using Octotree before the Pull Request update and there is one feature that I keep wanting, a way to see which files have been marked as **Viewed**! Right now, the only way to know if a file has been **Viewed** is to either click on it in the tree view, or scroll through the page which takes a long time given a large PR. 

It would be great if there would be a way to add a visual mark to the tree view to quickly determine if a file has been **Viewed** directly from the left tree view.

Thanks again!

**Comment by mkorsukov-optel** [comment]
In addition, please add tooltips for the file names. File tree is not resizable and file names can be quite long.

![image](https://github.com/community/community/assets/40869906/e5d3fc9c-3506-4ef4-b5c9-ad0f3ea3d7c9)

**Comment by mkorsukov-optel** [comment]
Regarding the "viewed state", it can be represented by the "eye" icon in green (viewed) and gray (not viewed) colors 😄

**Comment by boxfoot** [comment]
I would find this helpful, especially for long/large PRs.  It would be nice to have an option to hide files from the tree view if they've been marked "viewed" already so that I can easily jump among files that I haven't reviewed yet.

My typical approach for large PRs is to do an initial skim-through to get a high level understanding of the parts of the PR, and along the way I mark simple or boilerplate files as "viewed."  After this step I have maybe 20% of the files left that need a closer review and it would be nice to be able to clearly navigate among them.

**Comment by erri120** [comment]
This'd be super helpful for large and/or complex PRs where you jump around a lot.

**Comment by hanatharesh2712** [comment]
I also expect the same Tree view feature in GitDesktop as well. it will help for large PRs. Thank you!
